README.md

```diff
@@ -1 +1,24 @@
 # Operator Marketplace
+
+## Deploying the Marketplace Operator
+### Prerequisites
+In order to deploy the Marketplace Operator, you must:
+1. Be logged into an OpenShift cluster as a user with the RBAC permissions required to run this operator. These RBAC permissions are currently undefined, so log in as a user with the Cluster Admin Role.
+2. Have created the CatalogSourceConfig custom resource defined within the [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) project.
+3. Have authorization to pull the Marketplace Operator Image from quay.io/redhat/marketplace, you can make a request to join the “RedHat” organization by following the steps outlined [here](https://mojo.redhat.com/docs/DOC-1167707)
+4. Update the deploy/marketplace-pull-secret.yaml file with your secret.
+
+#### No Quay Access? Create the Marketplace Operator in your own public repo
+1. Update the deploy/operator.yaml to pull from a public registry. You should update the spec.template.spec.containers[0].image value with the <IMAGE DESTINATION> value. Example: `docker.io/awgreene/marketplace-operator`
+2. Remove the deploy/marketplace-pull-secret.yaml file to avoid errors.
+3. Build the Marketplace Operator with the following command.
+
+```bash
+$ operator-sdk build <IMAGE DESTINATION>
+$ docker push <IMAGE DESTINATION>
```

Comment:
We should re-think who the README is targeted at. A developer or someone who wants to try out marketplace. A user wanting to try out marketplace shouldn't have to build it. Plus given all developers won't have push access to the image specified in the deployment, we need to re-word this to reflect that. So splitting the README into a developer and user section will be useful.

Reply:
Document updated to distinctly separate steps to deploy the operator image vs. changing the operator.